src/@types/events.ts

```diff
@@ -1,5 +1,6 @@
 enum Event {
     ANNOTATION_CREATE = 'annotations_create',
+    ANNOTATION_REMOVE = 'annotations_remove',
```

Comment:
Should this be `annotations_remove_set` to indicate that this is an event to invoke an action in annotations just like the events that follow? `annotations_[create|delete|update]` are intended to be outgoing events to be consumed by client apps

Reply:
Hmm, that I'm not sure... because delete and update are going to handled through elements at this point and we will just emit remove and update/patch/something. `annotation_remove_set` doesn't make sense to me. How about something like `annotations_remove_id`?